airgun/views/rhai.py

```diff
@@ -0,0 +1,138 @@
+from widgetastic_patternfly import Button, Dropdown
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import (
+    Checkbox,
+    GenericLocatorWidget,
+    ParametrizedView,
+    Table,
+    Text,
+    TextInput,
+    View
+)
+
+from airgun.views.common import BaseLoggedInView
+
+
+class Overview(BaseLoggedInView):
+    title = Text(".//h1[@class='page-title']//span")
```

Comment:
why do you need title widget here?
Maybe you wanted to use it for:
```
    @property
    def is_displayed(self):
        return self.browser.wait_for_element(
            self.title, exception=False) is not None
```
Also, locator itself is a little bit ambiguous, so we forced to use a least preferable format, by text:
title = Text("//h1[text()='Overview']")